hack/verify-staging-imports.sh

```diff
@@ -79,7 +79,7 @@ if grep -rq '// import "k8s.io/kubernetes/' 'staging/'; then
 	exit 1
 fi
 
-for EXAMPLE in vendor/k8s.io/client-go/examples/{in-cluster,out-of-cluster,third-party-resources}; do
+for EXAMPLE in vendor/k8s.io/client-go/examples/{authenticate-in-cluster,authenticate-out-of-cluster,third-party-resources}; do
```

Comment:
I think authenticating is fine. The fact that you're pointing server URL/port is an implementation detail of authentication. The name change I made in this case was actually suggested by Jago (on the proposal doc) and I agree that it makes the point clear.

+ Helps with discovery as the letter A is listed first on GitHub.